cmake/dependencies/AMReX.cmake

```diff
@@ -86,7 +86,7 @@ option(pyAMReX_amrex_internal "Download & build AMReX" ON)
 set(pyAMReX_amrex_repo "https://github.com/AMReX-Codes/amrex.git"
     CACHE STRING
     "Repository URI to pull and build AMReX from if(pyAMReX_amrex_internal)")
-set(pyAMReX_amrex_branch "793ea9f717590d66d178a86298b82aac244b77a7"
+set(pyAMReX_amrex_branch "25.05"
```

Comment:
@atmyers 

Did you need that specific commit because of #435 or can we set `25.05` here to synchronize the releases?